Todo/UsersViewController.swift

```diff
@@ -0,0 +1,187 @@
+        
+        // Setup SearchController:
+        searchController = UISearchController(searchResultsController: nil)
+        searchController.searchResultsUpdater = self
+        searchController.searchBar.autocapitalizationType = .None
+        self.tableView.tableHeaderView = searchController.searchBar
+        
+        // Get username and database:
+        let app = UIApplication.sharedApplication().delegate as! AppDelegate
+        database = app.database
+        username = Session.username
+        
+        // Setup view and query:
+        setupViewAndQuery()
+    }
+    
+    override func viewWillAppear(animated: Bool) {
+        super.viewWillAppear(animated)
+        
+        // Setup navigation bar:
+        self.tabBarController?.title = taskList["name"] as? String
+        self.tabBarController?.navigationItem.rightBarButtonItem =
+            UIBarButtonItem(barButtonSystemItem: .Add, target: self, action: "addAction:")
+    }
+    
+    deinit {
+        if usersLiveQuery != nil {
+            usersLiveQuery.removeObserver(self, forKeyPath: "rows")
+            usersLiveQuery.stop()
+        }
+    }
+    
+    // MARK: - KVO
+    
+    override func observeValueForKeyPath(keyPath: String?, ofObject object: AnyObject?,
+        change: [String : AnyObject]?, context: UnsafeMutablePointer<Void>) {
+            if object as? NSObject == usersLiveQuery {
+                reloadUsers(usersLiveQuery.rows)
+            }
+    }
+    
+    // MARK: - UITableViewController
+    
+    override func tableView(tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return userRows?.count ?? 0
+    }
+    
+    override func tableView(tableView: UITableView,
+        cellForRowAtIndexPath indexPath: NSIndexPath) -> UITableViewCell {
+            let cell = tableView.dequeueReusableCellWithIdentifier("UserCell") as UITableViewCell!
+            let doc = userRows![indexPath.row].document!
+            
+            cell.textLabel?.text = doc["username"] as? String
+            
+            return cell
+    }
+    
+    override func tableView(tableView: UITableView,
+        editActionsForRowAtIndexPath indexPath: NSIndexPath) -> [UITableViewRowAction]? {
+            let delete = UITableViewRowAction(style: .Normal, title: "Delete") {
+                (action, indexPath) -> Void in
+                // Dismiss row actions:
+                tableView.setEditing(false, animated: true)
+                // Delete list document:
+                let doc = self.userRows![indexPath.row].document!
+                self.deleteUser(doc)
+            }
+            delete.backgroundColor = UIColor(red: 1.0, green: 0.23, blue: 0.19, alpha: 1.0)
+            return [delete]
+    }
+
+    
+    // MARK: - UISearchController
+    
+    func updateSearchResultsForSearchController(searchController: UISearchController) {
+        let text = searchController.searchBar.text ?? ""
+        let active = !text.isEmpty
```

Comment:
Again, I think it's clearer to use `if text.isEmpty {...} else {...}` instead of multiple `?:`s.